_config.yml

```diff
@@ -121,6 +121,53 @@ navigation:
           - text: Travel guide (table of contents)
             url: travel-guide-table-of-contents/
             internal: true
+          - text: Out of office expectations
+            url: leave/#leave-checklist
+            internal: false
+  - text: Performance management
```

Comment:
I know there's only one page under this right now, but I'm going ahead and stubbing it out because
1. It's a thing people are highly interested in, so it should be less buried.
2. We're working on more comprehensive docs -- a playbook -- based on our 2019 review retro, so it gives a good spot for those efforts.